content/200-concepts/300-more/050-environment-variables/index.mdx

```diff
@@ -7,7 +7,7 @@ tocDepth: 2
 
 <TopBlock>
 
-You can use environment variables to provide configuration options when a CLI command is invoked. Environment variables allow you to:
+You can use environment variables to provide configuration options when a CLI or Client command is invoked. When CLI commands are used, environment variables allow you to:
```

Comment:
This is also where we should probably have a section how to set the environment variables on your machine! We mention "machine-specific variables" below, but only explain how to use the file. But the file is just a better variant of setting it manually (Which many people will do or have to do even).